airflow/cli/commands/dag_command.py

```diff
@@ -123,6 +124,7 @@ def dag_backfill(args, dag=None):
                 end_date=args.end_date,
                 confirm_prompt=not args.yes,
                 include_subdags=True,
+                dag_run_state=State.NONE,
```

Comment:
Do I correctly understand that previously the default value `State.RUNNING`? If yes then we should add note in UPDATING.md.

Reply:
The backfill experience from the user side should be almost the same. If user backfill with `--reset-dagruns`, it will first clear the DagRun (setting the state to None) then the backfill scheduler would set the state to `RUNNING`.